### PR TITLE
Feature/seeds

### PR DESCRIPTION
**Pull request by Brainybrian316**
1. added seeds for user
2. added anticipated seeds for projects `can possibly set seeds to be more customizable... ex from user seeds`
3. commented out unnecessary field from project.
4. changed project schema name to correct export/utilize the model

### CODE REVIEW

**Comment by Brainybrian316** [comment]
* removed fields from connect due to it not being supported anymore.
* added `npm run seeds` in packages  scripts
* added `npm run develop` to only run backend server in packages scripts
* changed datatypes to strings in documentation for mongo text was not a type.. [https://www.tutorialspoint.com/mongodb/mongodb_datatype.htm](url)

**Comment by Brainybrian316** [comment]
changed regex in user to require
Minimum eight characters, at least one letter and one number:

File: src/services/config/connection.js [hunk 1]
```diff
@@ -1,10 +1,6 @@
 const mongoose = require('mongoose');
 
 mongoose.connect(process.env.MONGODB_URI || 'mongodb://localhost/devise', {
-  useNewUrlParser: true,
-  useUnifiedTopology: true,
-  useCreateIndex: true,
-  useFindAndModify: false
```
Comment (cobalt88) [inline src/services/config/connection.js:7]: Don't we need these?

Comment (Brainybrian316) [reply src/services/config/connection.js:7]: i thought so but when i included them i kept getting errors that they are depreciated.

File: src/services/models/DeviseProjects.js [hunk 2]
```diff
@@ -11,11 +12,11 @@ const DeviseProjectsSchema = new Schema({
   },
   //description is the detailed description of the project, displayed on the project page.
   description: {
-    type: Text,
```
[the remark this answers is not in the document]
Comment (Brainybrian316) [reply src/services/models/DeviseProjects.js:14]: plus there no other data type to change it too just review the link of datatypes that i provided if anything but when i seeded the data its a paragraph of 10 sentences and it looked fine on the db

**Pull request merged**